core-bundle/contao/dca/tl_module.php

```diff
@@ -73,7 +73,7 @@
 		'booknav'                     => '{title_legend},name,headline,type;{nav_legend},showProtected,showHidden;{reference_legend:hide},rootPage;{template_legend:hide},customTpl;{protected_legend:hide},protected;{expert_legend:hide},cssID',
 		'articlenav'                  => '{title_legend},name,headline,type;{config_legend},loadFirst;{template_legend:hide},customTpl;{protected_legend:hide},protected;{expert_legend:hide},cssID',
 		'sitemap'                     => '{title_legend},name,headline,type;{nav_legend},showProtected,showHidden;{reference_legend:hide},rootPage;{template_legend:hide},customTpl,navigationTpl;{protected_legend:hide},protected;{expert_legend:hide},cssID',
-		'login'                       => '{title_legend},name,headline,type;{config_legend},autologin;{redirect_legend},jumpTo,redirectBack;{template_legend:hide},customTpl;{protected_legend:hide},protected;{expert_legend:hide},cssID',
+		'login'                       => '{title_legend},name,headline,type;{config_legend},autologin;{redirect_legend},jumpTo,pwResetJumpTo,redirectBack;{template_legend:hide},customTpl;{protected_legend:hide},protected;{expert_legend:hide},cssID',
```

Comment:
```suggestion
		'login'                       => '{title_legend},name,headline,type;{config_legend},autologin;{redirect_legend},jumpTo,redirectBack,pwResetJumpTo;{template_legend:hide},customTpl;{protected_legend:hide},protected;{expert_legend:hide},cssID',
```

The `redirectBack` checkbox has a relation to `jumpTo`, so they probably should stay together. Not sure what the UI will look like though.

Reply:
It would look like this:
![image](https://github.com/contao/contao/assets/55794780/fa59f931-fde9-4e5e-b414-a2243842a901)

We could make use of w50 cbx m12 to have it next to the selection if you say that they should stay together:
![image](https://github.com/contao/contao/assets/55794780/f7fd8b4d-b9de-41c3-86d7-912f86809cb8)
![image](https://github.com/contao/contao/assets/55794780/e2dee426-863b-4248-a0df-d392858731d7)

Wdyt?